src/components/Registration/RegAddEditOrgPerson.vue

```diff
@@ -164,6 +164,15 @@
                   a bank or a railway, use the manual entry form. All other types of business cannot
                   be {{ isProprietor ? 'the Proprietor' : 'a partner' }}.
                 </p>
+
+                <HelpContactUs class="mt-6" />
```

Comment:
This might need `v-if="!isRoleStaff"`. Check with UI/UX.

Reply:
Yui confirmed it can stay for staff.